lucupy/minimodel/atom.py

```diff
@@ -30,9 +30,9 @@ class Atom:
 
     """
     id: int
-    exec_time: timedelta
-    prog_time: timedelta
-    part_time: timedelta
+    exec_time: timedelta = field(hash=False, compare=False)
```

Comment:
I assuming we only want to hash and compare by id.

Reply:
I was thinking about that, too, and I think it makes sense. I don't think we ever actually use any of these as a key and only as a value, so we never hash on them, but I figured at least ensure the mutable parts are left out all the same.